src/app/address/controllers/address/manual.js

```diff
@@ -125,6 +138,19 @@ class AddressController extends BaseController {
 
     return hasChanged !== -1;
   }
-}
 
+  defaultToFirstField(formFields, first, req) {
+    formFields[first].validate.push({
+      fn: ukBuildingAddressEmptyValidator,
+      message: req.translate("validation.houseNameOrHouseNumber"),
+    });
+  }
+
+  isBuildingAddressEmpty(req) {
```

Comment:
This is duplicated from the nonUK controller, right? Do you think we could extract it into a helper?

Reply:
Oh the field name is different. Hmmmm these field names are needed for translations, right? So we can't just align them?